Helper/Config.php

```diff
@@ -2509,4 +2509,37 @@ public function getMerchantDashboardUrlFromAdditionalConfig($storeId = null)
     {
         return $this->getAdditionalConfigProperty('merchantDashboardURL', $storeId);
     }
+    
+    /**
+     * Get Integration base URL
+     *
+     * @param null|string $storeId
+     *
+     * @return string
+     */
+    public function getIntegrationBaseUrl($storeId = null)
+    {
+        //Check for sandbox mode
+        if ($this->isSandboxModeSet($storeId)) {
+            return $this->getMerchantDashboardUrlFromAdditionalConfig($storeId) ?: $this->getCustomURLValueOrDefault(self::XML_PATH_CUSTOM_API, self::API_URL_SANDBOX);
+        }
+        return self::API_URL_PRODUCTION;
+    }
+    
+    /**
+     * Get Bolt additional configuration for integration base URL, stored in the following format:
+     *
+     * {
+     *   "integrationBaseURL": "https://api-sandbox.bolt.com/"
+     * }
+     * defaults to empty string if not set
+     *
+     * @param int|string $storeId
+     *
+     * @return string
+     */
+    public function getIntegrationBaseUrlFromAdditionalConfig($storeId = null)
```

Comment:
Looks like we don't use this helper and can just delete it.

Reply:
Removed